tests/test_rtstruct_builder.py

```diff
@@ -164,6 +165,36 @@ def test_contour_data_sizes(new_rtstruct: RTStruct):
     # Then using approximation leads to less data within the contour data
     assert get_data_len_by_index(new_rtstruct, 0) < get_data_len_by_index(new_rtstruct, 1)
 
+def test_load_sorted_series(oriented_series_path):
```

Comment:
Curious, what are we testing here?

Reply:
It was checking if we can determine ImagePosition's of all the slices using ImageOrientation and ImagePosition's of the first and last slice.

Now that I think about it this test is unnecessary, because we check the correctness of ImageOrientation inside get_slice_directions anyway. Removed it in the next commit.